wolox-books/src/app/screens/auth/screens/book-list/book-list.component.html

```diff
@@ -1,75 +1,26 @@
-<app-navbar></app-navbar>
 <div class="container">
+  <div class="search-bar">
+    <input
+      #searchBar
+      type="text"
+      class="search-bar__input"
+      placeholder="Buscar por titulo del libro..."
+      (keyup)="onKey(searchBar.value)"
```

Comment:
Why not using an ngModel and a small form?

Reply:
Done, great solution thanks! 😎